lms/djangoapps/discussion/static/discussion/js/views/discussion_user_profile_view.js

```diff
@@ -0,0 +1,92 @@
+(function(define) {
+    'use strict';
+
+    define([
+        'underscore',
+        'jquery',
+        'backbone',
+        'gettext',
+        'URI',
+        'edx-ui-toolkit/js/utils/html-utils',
+        'common/js/components/utils/view_utils',
+        'common/js/discussion/discussion',
+        'common/js/discussion/utils',
+        'common/js/discussion/views/discussion_thread_profile_view',
+        'text!discussion/templates/user-profile.underscore',
+        'text!common/templates/discussion/pagination.underscore'
+    ],
+        function(_, $, Backbone, gettext, URI, HtmlUtils, ViewUtils, Discussion, DiscussionUtil,
+                 DiscussionThreadProfileView, userProfileTemplate, paginationTemplate) {
+            var DiscussionUserProfileView = Backbone.View.extend({
+                events: {
+                    'click .discussion-paginator a': 'changePage'
```

Comment:
This is legacy code that was moved, we won't be making any fixes to it on this branch